src/seedu/addressbook/common/PersonNameComparator.java

```diff
@@ -0,0 +1,11 @@
+package seedu.addressbook.common;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.Comparator;
+
+public class PersonNameComparator implements Comparator<ReadOnlyPerson>{
```

Comment:
This class and its non-trivial methods should have header comments. 
If not, developers who write client code for this class will not know how to use it.